### DISCUSSION TITLE
possibility to delete all pools except ...

### DISCUSSION THREAD

**Discussion by themark147** [symfony/symfony#51023] [Q&A] [answered]
Hey community

is there any possibility to remove all pools except one or two pools?
I couldnt find any normal solution

and removing pools one by one except that I dont want to invalidate seems quite odd to me

Or I have to create some own custom cmd?

thanks :pray:

**Comment by MatTheCat** [comment, marked as answer]
Hello, this is not supported yet but I opened #51058.

In the meantime you’d have to create your own command.

**Reply by themark147** [reply]
wau thats nice to see someone else need that feature as well :hugs: 

thanks It will be fine to see that feature in 6.4
